src/System.Data.Common/src/System/Data/Filter/BinaryNode.cs

```diff
@@ -1096,7 +1096,7 @@ private object EvalBinaryOp(int op, ExpressionNode left, ExpressionNode right, D
 
                             if ((vRight == DBNull.Value) || (right.IsSqlColumn && DataStorage.IsObjectSqlNull(vRight)))
                                 continue;
-                            Debug.Assert((!DataStorage.IsObjectNull(vLeft)) && (!DataStorage.IsObjectNull(vRight)), "Imposible..");
+                            Debug.Assert((!DataStorage.IsObjectNull(vLeft)) && (!DataStorage.IsObjectNull(vRight)), "Impossible.");
 
```

Comment:
Snuck in one more `Debug.Assert` here... Turns out that was the last one! Good job Microsoft! A through Z string literal check complete.

Reply:
Large-scale spellchecking was done before, but some got missed and some got added since.